src/_P118_Itho.ino

```diff
@@ -593,4 +804,56 @@ void PLUGIN_118_PluginWriteLog(const String& command)
   addLogMove(LOG_LEVEL_INFO, log);
 }
 
+void PLUGIN_118_PluginSetDestIDSrcID(struct EventStruct *event, uint8_t (& srcID) [3], uint8_t (& destID) [3], char(& tmpTmpID) [9])
```

Comment:
Why are you using a reference here?
arrays as function parameters are always handed over as a pointer.
So no idea what C++ will actually make of this, but I've got a feeling this is probably not doing what one may expect at first.

Reply:
Well let's say that I used bruteforced/googled through a pointer and array nightmare :).... which in the end just is a nice way of saying "i don't know" to your question :)

Reply:
OK, good to know, so we also need to look for other potential programming issues, like checking for null-pointer dereferences.
And don't worry, my programming skils related to JavaScript suddenly appear to be lost when StackOverflow is down.
So I can relate to this :)